.one-pipeline-cd.yaml

```diff
@@ -64,12 +69,19 @@ deploy:
     for INVENTORY_ENTRY in $(echo "${INVENTORY_ENTRIES}" | jq -r '.[] '); do
       APP=$(cat "${INVENTORY_PATH}/${INVENTORY_ENTRY}")
       ARTIFACT=$(echo "${APP}" | jq -r '.artifact')
+      echo "${ARTIFACT}"
+
+      if [[ ${ARTIFACT} == *"icr.io/appcafe/open-liberty/samples/"* ]];then
+        echo "Exclude samples from publishing: ${ARTIFACT}"
+        continue
+      fi
+
       NAME=$(echo "${APP}" | jq -r '.name')
       DIGEST=$(echo "${APP}" | jq -r '.sha256' )
       TYPE=$(echo "${APP}" | jq -r '.type' )
       REPO=$(echo "${APP}" | jq -r '.repository_url' ).git
       COMMIT=$(echo "${APP}" | jq -r '.commit_sha' )
-      echo "${ARTIFACT}"
+
```

Comment:
Was this an intentional delete?

Reply:
It's moved up (above the new if-condition)